internal/Picker/tests/Picker-specs.js

```diff
@@ -123,6 +123,30 @@ describe('Picker Specs', () => {
 		expect(actual).toBe(expected);
 	});
 
+	test('should register onKeyDown event for keyDown \'right arrow\' on increment button', () => {
+		const handleChange = jest.fn();
+		render(
+			<Picker index={0} max={6} min={0} onKeyDown={handleChange} step={3} value={0} />
+		);
+		const picker = screen.getAllByRole('button')[0];
+
+		fireEvent.keyDown(picker, {key: "right arrow", keyCode: 39, code: "right arrow", charCode: 39});
```

Comment:
Let's added keyDown functions like Slider-spec.
```suggestion
		rightKeyDown(picker);
```
and also please added in line 8.
```
const increment = (picker) => userEvent.click(picker.firstElementChild);
const decrement = (picker) => userEvent.click(picker.lastElementChild);
const keyDown = (keyCode) => (picker) => fireEvent.keyDown(picker, {keyCode});

const leftKeyDown = keyDown(37);
const rightKeyDown = keyDown(39);
```

Reply:
Done